src/StoreKeeper/WooCommerce/B2C/Helpers/DateTimeHelper.php

```diff
@@ -2,10 +2,39 @@
 
 namespace StoreKeeper\WooCommerce\B2C\Helpers;
 
+use DateTime;
+use DateTimeZone;
+use Exception;
 use StoreKeeper\WooCommerce\B2C\I18N;
 
 class DateTimeHelper
 {
+    const MYSQL_DATE_FORMAT = 'Y-m-d H:i:s';
+
+    /**
+     * Get current date and time
+     * Don't use non-GMT time, unless you know the difference and really need to.
+     *
+     * @throws Exception
+     */
+    public static function currentDateTime($gmt = true): DateTime
+    {
+        $timezone = $gmt ? new DateTimeZone('UTC') : wp_timezone();
+
+        return new DateTime('now', $timezone);
+    }
```

Comment:
```suggestion
    public static function currentDateTime(): DateTime
    {
        return new DateTime('now', new DateTimeZone('UTC'));
    }
```
It's never used with `$gmt = false` -> dead code.

Reply:
Removed dead code.